playbook-website/config/menu.yml

```diff
@@ -572,3 +572,8 @@ kits:
       with avatar, titles, name and territory. This is a versatile kit with features
       than can be added to display more info.
     status: stable
+- category: link
+  components:
+    - name: link
+      platforms: all
+      status: beta
```

Comment:
I don't think 'Link' should be its own category, it needs to be nested under a parent. My assumption is 'Typography' but will let @jasperfurniss weigh in there!

Reply:
I'll move it to "Typography" for now.

Reply:
Should I add a description too? Or should we add a description when this moves from beta to stable?

Reply:
I'll leave a description out since it's just in beta right now.